main.py

```diff
@@ -0,0 +1,174 @@
+from chess_utils import *
+from teams import Team, get_score_difference
+import timer
+import pygame
+import screen
+import bot
+import os
+import colors
+
+SOUNDS_PATH = 'sounds'
+
+team_got_turn = None
+team_doesnt_got_turn = None
+turn_number = 0
+
+
+def redraw_game_screen():
+    screen.draw_bg(team_got_turn, team_doesnt_got_turn)
+
+    for line in screen.squares:
+        for square in line:
+            if square.current_piece is not None:
+                square.current_piece.draw()
+
+    pygame.display.flip()
+
+
+def get_square_clicked():
+    mouse_pos = pygame.mouse.get_pos()
+
+    for line in screen.squares:
+        for square in line:
+            if square.rect.collidepoint(mouse_pos):
+                return square
+
+
+def switch_turn(white_team, black_team):
+    global team_got_turn
+    global team_doesnt_got_turn
+    team_got_turn = team_doesnt_got_turn
+    if team_got_turn is white_team:
+        team_doesnt_got_turn = black_team
+    else:
+        team_doesnt_got_turn = white_team
+
+    timer.switch_timers(team_got_turn, team_doesnt_got_turn)
+
+
+def remove_eaten_pieces(white_team, black_team):
+    for piece in white_team.pieces + black_team.pieces:
+        piece_team = white_team if piece.team.is_white_team else black_team
+        if piece.is_eaten:
+            piece_team.pieces.remove(piece)
+
+
+def update_game_after_move(piece_clicked, black_team, white_team):
+    redraw_game_screen()
+    global turn_number
+    pygame.mixer.Sound(os.path.join(SOUNDS_PATH, 'pong.wav')).play()
+
+    switch_turn(white_team, black_team)
+
+    print(piece_clicked)
+
+    if is_checkmated(team_got_turn, team_doesnt_got_turn):
+        screen.draw_winner(team_doesnt_got_turn)
+        raise Checkmated
+
+    if is_tie(team_got_turn, team_doesnt_got_turn):
+        text = f"Tie"
+        text_surface = screen.LARGE_FONT.render(text, False, colors.DARK_GREEN)
```

Comment:
Export print tie to screen module in function named print_tie.

Reply:
:+1: